### PR TITLE
waypipe: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
I welcome any other description. "A transparent proxy for Wayland applications" is far from newbie friendly language.

File: pages/linux/waypipe.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# waypipe
+
+> A transparent proxy for Wayland applications.
```
Comment (vitorhcl) [inline pages/linux/waypipe.md:3]:
```suggestion
> Remotely run graphical applications under a Wayland compositor.
```
How about this?

File: pages/linux/waypipe.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# waypipe
+
+> Remotely run graphical applications under a Wayland compositor.
+> More information: <https://gitlab.freedesktop.org/mstoeckl/waypipe>.
+
+- Run a graphical program remotely and display it locally:
+
+`waypipe ssh {{user}}@{{server}} {{program}}`
+
+- Open an ssh tunnel to run any program remotely and display it locally:
```
Comment (fazlearefin) [inline pages/linux/waypipe.md:10]:
```suggestion
- Open a ssh tunnel to run any program remotely and display it locally:
```

Comment (Managor) [reply pages/linux/waypipe.md:10]: It's pronounced "ess-ess-aitch", thus "an" is correct

Comment (vitorhcl) [inline pages/linux/waypipe.md:10]:
```suggestion
- Open an SSH tunnel to run any program remotely and display it locally:
```

**Review by vitorhcl — APPROVED** [review]
LGTM after my suggestion

**Review by fazlearefin — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**